examples/static-batch-test.py

```diff
@@ -69,9 +68,9 @@ def generator_config():
     )
 
 def discriminator_config():
-    return hg.discriminators.pyramid_discriminator.config(
+    return hg.discriminators.autoencoder_discriminator.config(
```

Comment:
Some issue as 2d-trainer - we might want to search for something other than BEGAN.  Shouldn't hold up the release.